tools/mti-report/mti_reports.xml

```diff
@@ -0,0 +1,85 @@
+<tool id="mti_reports" name="MTI Analysis and QC Reports" version="">
+    <description>Automated generation of HTML analysis and QC reports for Galaxy MTI workflows</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+        <![CDATA[
+        #if $reports.selected_report == "QC":
+        python '$__tool_directory__/QC/cycle_concordance.py'
+            --file '$registered_image'
+            --markers '$markers'
+            --pattern '$pattern' &&
+        
+        python '$__tool_directory__/QC/segmentation_thumbnails.py'
+            --file '$registered_image'
+            --segmentation '$segmentation'
+            --channel '$segmentation_channel' &&
+
+        ln -s '$anndata' ./adata.h5ad &&
+
+        python '$QC_script' &&
+
+        ln -s '$__tool_directory__/QC/QC_template.md' ./QC.md &&
+
+        python -m markdown ./QC.md > QC_report.html
+        #end if
+        ]]>
+    </command>
+    <configfiles>
+        <configfile name = "QC_script">
+import os
+import sys
+import re
+import numpy as np
+import pandas as pd
+import anndata as ad
+import plotly.graph_objects as go
+import matplotlib.pyplot as plt
+
+from '$__tool_directory__/MTIreport' import quality
```

Comment:
I'm skeptical this works in all setups. Should this tool be packaged instead and installed with conda?